composer.json

```diff
@@ -13,7 +13,7 @@
         "doctrine/inflector":   "~1.1",
         "doctrine/orm":         "~2.4",
         "symfony/filesystem":   "^3.0.2|~2.5|^4.0",
-        "twig/twig":            "^2.0||^1.28"
+        "twig/twig":            "~2.7"
```

Comment:
`^2.7.0` and this means we need to do a major release.

Reply:
Disagree on ^2.7. Twig does not follow semver. 

Also from this package perspective this is not a BC-break. It just means that if you do not update twig, you can also not update this package.

Reply:
It should follow semver (at least to from what I've seen/heard/read), but I cannot find anything to make a valid stance, so :+1:.

Reply:
Can't we keep the 1.x branch in here as well?

The security issue was fixed in 1.38.0.